ansible/templates/centos7/deps_ice3.6.sh.j2

```diff
@@ -9,4 +9,7 @@ set -e -u -x
 {% endif %}
 
 # option 1 and option 2
-{{ virtualenv_path }}/bin/pip install --upgrade https://github.com/ome/zeroc-ice-py-centos7/releases/download/0.1.0/zeroc_ice-3.6.4-cp27-cp27mu-linux_x86_64.whl
+. {{ virtualenv_path }}/bin/activate
+pip install --upgrade pip
+
+pip install --upgrade https://github.com/ome/zeroc-ice-py-manylinux/releases/download/0.1.0/zeroc_ice-3.6.5-cp36-cp36m-manylinux2010_x86_64.whl
```

Comment:
As discussed in slack this includes a static openssl library so shouldn't be used in production. If this is for testing only and not for the production docs this is fine.